statusHandler/statusMetricsProvider.go

```diff
@@ -284,28 +301,20 @@ func (sm *statusMetrics) NetworkMetrics() map[string]interface{} {
 
 // must be called under mutex protection
 func (sm *statusMetrics) loadUint64Metric(metric string) uint64 {
```

Comment:
this function became a little bit useless. we can just call `xxxxvalue := sm.uint64Metrics[metric]` as the uint64/int64 defaults are 0 and the string default is ""

Valid also for `loadStringMetric`

Reply:
done, removed